src/frontend/src/pages/CreateChangeRequestPage/CreateChangeRequestView.tsx

```diff
@@ -122,6 +124,8 @@ const CreateChangeRequestsView: React.FC<CreateChangeRequestViewProps> = ({
     });
   });
 
+  wbsDropdownOptions.sort(wbsNumComparator);
```

Comment:
You need to pass in a method into sort like this: .sort((a,b) => comparator(a.id, b.id) so you don't have to take out the ids in the comparator